service/controller/resource/tccp/create.go

```diff
@@ -12,8 +15,8 @@ import (
 	"sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
 
 	"github.com/giantswarm/aws-operator/pkg/awstags"
+	pkgtemplate "github.com/giantswarm/aws-operator/pkg/template"
```

Comment:
I think this is not the best we can do. Check the other resources how we do it there. We already import a `template` package, so doing it again smells and there should be a better solution.

Reply:
When you want to address this as a next step please file a TODO issue.